### PR TITLE
Updated mixer config to bring in latest changes

### PR DESCRIPTION
**Pull request by nrjpoddar**
Updated to get to 31948644cc53bb8569e4073486d7796540909adc from istio/istio
master branch.

### CODE REVIEW

File: istio-telemetry/mixer-telemetry/templates/config.yaml [hunk 1]
```diff
@@ -511,7 +511,7 @@ spec:
       destination_principal: destination.principal | "unknown"
       destination_app: destination.labels["app"] | "unknown"
       destination_version: destination.labels["version"] | "unknown"
-      destination_service: destination.service.host | request.host | "unknown"
+      destination_service: destination.service.host | "unknown"
```
Comment (nrjpoddar) [inline istio-telemetry/mixer-telemetry/templates/config.yaml]: Note that this is an extra diff that was outdated in installer. @howardjohn

File: istio-telemetry/mixer-telemetry/templates/config.yaml [hunk 2]
```diff
@@ -542,7 +542,7 @@ spec:
...
```
Comment (nrjpoddar) [inline istio-telemetry/mixer-telemetry/templates/config.yaml]: Note that this is an extra diff that was outdated in installer. @howardjohn

**Comment by nrjpoddar** [comment]
I am not confident of the second commit that I added after running `make gen`.

**Comment by nrjpoddar** [comment]
> I am not confident of the second commit that I added after running `make gen`.

Scratch that, after regenerating them with `BUILD_WITH_CONTAINER=1`, change set looks more sane.

**Review by howardjohn — APPROVED** [review]

**Comment by howardjohn** [comment]
This does not need a 1.3 backport here, this was super experimental in 1.3 and has tons of bugs already

**Comment by howardjohn** [comment]
Thanks!

**Pull request merged**